charts/kubernetes-policy-controller/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
Mhh... you're specifying OPA v.0.6 and kubernetes-policy-controller v.1.2 in your default values.yaml.